spec/spec_helper.rb

```diff
@@ -118,24 +120,6 @@
   end
 end
 
-# Clusters and saves each element in an array of clusterables
-# DRY for the many times the tests need to do something like:
-# Clustering::ClusterXYZ.new(xyz).cluster.tap(&:save)
-def cluster_tap_save(*clusterables)
```

Comment:
This isn't going to work as-is; eventually, we might find that we want a different approach to DRYing out setting up fixture data.

Reply:
It's probably at the place where we could -- i.e. there are enough examples to figure out what exactly we don't want to repeat, and it might save some thinking in the specs if there were a generalized "persist" helper function that could do whatever it needed to do.